### ISSUE TITLE
[Feature Request] Menu activator open on enter key event

### ISSUE THREAD

**Issue by frattaro** [vuetifyjs/vuetify#5386]
### What problem does this feature solve?
Accessibility
Adhere to specs

### What is your proposed solution?
Needs openOnEnter bool prop added to VMenu and events attached in activator mixin: https://github.com/vuetifyjs/vuetify/blob/dev/src/components/VMenu/mixins/menu-activator.js

**Comment by jacekkarczmarczyk** [comment]
You can open menu using eny event you want
https://codepen.io/anon/pen/bPNvMo?editors=1000
Not sure if we need another prop for this

**Comment by frattaro** [comment]
Not sure where my head was at 6 months ago when I wrote this, but I believe the default should be to open on enter, as that is what would be expected from an accessibility standpoint. See: https://material-components.github.io/material-components-web-catalog/#/component/menu

**Comment by KaelWD** [comment]
If you change @jacekkarczmarczyk's example to `v-on="scope.on"` then it already does open on enter by default.

**Comment by jacekkarczmarczyk** [comment]
And it's because for buttons enter triggers the click event

**Comment by frattaro** [comment]
Oh I see. There's an `if (this.openOnClick...` statement in the `menu-activator` mixin. I was definitely thinking "whatever the implementation, default accessibility keyboard inputs should be treated as equals to the mouse inputs"

**Comment by jacekkarczmarczyk** [comment]
As far as I understand this issue can be closed, reopen if I'm wrong

**Comment by frattaro** [comment]
I'm not sure how, in any way, that solution addresses the issue.

On the roadmap, accessibility is stated as a goal of v2 and does not mesh with a "working as intended" resolution.

**Comment by jacekkarczmarczyk** [comment]
In a way that it opens menu on Enter?

**Comment by frattaro** [comment]
My understanding is that the expected result of an accessible component is an equivalent user experience across ability levels. Whether that means adding activate on keyboard enter as default or removing click to activate as default is an implementation decision.

**Comment by jacekkarczmarczyk** [comment]
> Whether that means adding activate on keyboard enter as default

We can't add again something that we have already added and never removed.

> removing click to activate as default is an implementation decision

Remove click doesn't sound like making the component more accessible, I'm not the accessibility expert though and I'd love to hear what would be the use case 

I'll ask differently - do you have a use case where 
1. you are not able to achieve what you want using slot's scope or 
2. slot's scope works but you just prefer using props or 
3. enter doesn't open the menu when you attach default event handlers (`<tag v-on="on">`)? 

If 1. or 3. please provide a codepen

**Comment by frattaro** [comment]
As of 1.3.x (when this ticket was opened), the behavior mismatch was the case with the old slot syntax. See: https://codepen.io/frattaro/pen/wLGdom

In vuetify@1.4.8, vuetify@latest, looks fixed, still supports old slot syntax.

In vuetify@next, old slot syntax throws error:
[Vuetify] The activator slot must be bound, try '<template v-slot:activator='{ on }'><v-btn v-on='on'>'

Maybe the issue was fixed when the new slot syntax support was added? Interesting that in vuetify@next there is no auto wire-up.
